bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/thing/ThingTypeResource.java

```diff
@@ -91,7 +92,7 @@ protected void unsetConfigDescriptionRegistry(ConfigDescriptionRegistry configDe
 
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = "Gets all available things types.", response = ThingTypeDTO.class, responseContainer = "Set")
+    @ApiOperation(value = "Gets all available things types without config parameters, channels and properites.", response = ThingTypeDTO.class, responseContainer = "Set")
```

Comment:
The Swagger documentation still claims that you are returning ThingTypeDTOs, which might be a bit misleading (technically, it is of course right, but the user might expect that the values are provided and not null).
I wonder if we should introduce something like "StrippedDTOs" in addition to our "EnrichedDTOs" that make it more explicit, which information is returned. wdyt?